ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -249,14 +267,12 @@ export default defineComponent({
       areAllSelected: computed((): boolean => {
         return localState.results?.every(result => result && result.selected === true)
       }),
-      isIndeterminate: computed((): boolean => {
+      isIndeterminate: computed(() => {
         return localState.selectAll !== localState.areAllSelected
       }),
-      areAllLienSelected: computed((): boolean => {
-        return localState.results?.every(result => result && result.includeLienInfo === true)
-      }),
       isLienIndeterminate: computed((): boolean => {
```

Comment:
the indeterminate state aren't used but I kept it because it was there originally, and we might implement mix mode later

Reply:
Hmm, what do you mean by not being used. 
If a user uses Select All Lien but then manually de-selects 1 of those, that would result in an indeterminate state. That was current implementation, is that no longer functional you mean?

Reply:
if the user checks "selects all Lien" then uncheck one, then the "select all Lien" will just get unchecked, there's no indeterminate state so to speak.

Reply:
Is that by design or intention? 
Previous requirements were for that state to be included so i would double check with Tracey about removing it. 

Current in Dev:
![Screen Shot 2022-08-04 at 3 38 43 PM](https://user-images.githubusercontent.com/53542131/182964738-f7f4f974-9bb0-4406-a8cb-acf5869406d3.png)

Reply:
![image](https://user-images.githubusercontent.com/62360407/182964883-3b7c9abf-d935-48a0-886a-8094a0899c7b.png)

Reply:
Ok, thanks for clarifying! 
It helps to note these things in the PR or in a comment :)